test/JDBC/expected/Test-sp_rename-vu-prepare.out

```diff
@@ -8,6 +8,12 @@ GO
 CREATE TABLE sp_rename_vu_table2(sp_rename_vu_t2_col1 int, sp_rename_vu_t2_col2 int);
 GO
 
+SET quoted_identifier ON
+GO
+
+CREATE TABLE sp_rename_vu_table_delim("sp_rename_vu_td1_col1" int, [sp_rename_vu_td1_col2] int);
```

Comment:
Please refer to the test guidelines and add test cases for upper, lower and mixed-case